### PR TITLE
ctc using nn.Dropout; ds2 libri vald batch_size / 4

### PR DESCRIPTION
**Pull request by zh794390558**
### PR types

Others
### PR changes

Others
### Describe

ctc using nn.Dropout; ds2 libri vald batch_size / 4

### CODE REVIEW

**Review by Jackwaterveg — APPROVED** [review]
LGTM
